### ISSUE TITLE
The stable docs link in the README is broken

### ISSUE THREAD

**Issue by LilithHafner** [LuxDL/DocumenterVitepress.jl#72]
The readme link: https://luxdl.github.io/DocumenterVitepress.jl
301 Redirects to https://luxdl.github.io/DocumenterVitepress.jl/
which 200 redirects to https://luxdl.github.io/DocumenterVitepress.jl/stable/
which returns a webpage with status code 200, but this is the webpage:

![Webpage showing 404 error](https://github.com/LuxDL/DocumenterVitepress.jl/assets/60898866/f20d4979-54b7-45b8-87e2-fe89ff138357)

**Comment by asinghvi17** [comment]
Yeah I think Vitepress is not happy with the symlink.  Will investigate...

**Comment by asinghvi17** [comment]
Yeah I think Vitepress is not happy with the symlink.  Will investigate...

**Comment by lazarusA** [comment]
oh... this is bad. I didn't noticed this extra step. IMO, we shouldn't mess with it, all because we want back the old pattern. And if at all, it should be optional, moving forward we should have the `cleanUrls` as given by vitepress.

**Comment by asinghvi17** [comment]
I guess we can provide a fixer function in `deploydocs` like so:

https://github.com/JuliaDocs/Documenter.jl/blob/a6f6c0c7e33f4b0c3fab38811ac691d646e2d46c/src/deploydocs.jl#L273-L276

which solves these problems?  Not sure if that applies itself at the right time, though.
